### DISCUSSION TITLE
[@types/geojson]: fixing limitation of 2/3 numbers per Position

### DISCUSSION THREAD

**Discussion by tupe12334** [DefinitelyTyped/DefinitelyTyped#71441] [Issues with a @types package]
According to the spec of the interface of what is a Position: https://datatracker.ietf.org/doc/html/rfc7946#section-3.1.1
```text
A position is an array of numbers.  There MUST be two or more
   elements.  The first two elements are longitude and latitude, or
   easting and northing, precisely in that order and using decimal
   numbers.  Altitude or elevation MAY be included as an optional third
   element.
```
Currently, the Position type supports as many numbers as the user wants, we need to change it to be between two or three.

I know that there was a dissection about that a year ago and 6 years ago, but its I think its time to fix that, as of today its the standard to use two or three values
Link to old discussion about this topic
https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/67773

[a comment by an automated account is not shown]

**Comment by icholy** [comment]
There's nothing to fix. You're proposing a massive breaking change with marginal value. Hard "NO".
